### ISSUE TITLE
Resolve apply for rule services in host services view

### ISSUE THREAD

**Issue by friesoft** [Icinga/icingaweb2-module-director#1868]
## Expected Behavior
The services view of a host should show a row for every object generated by an apply for rule

Currently (single entry named like the apply rule object):
> Apache Prozess (apply for host.vars.httpds) (assign where "linux" in host.groups)

Expected (multiple entries named like the generated objects):
> Apache Prozess /opt/mein-erster-apache/bin/httpd (apply for host.vars.httpds) (assign where "linux" in host.groups)
> Apache Prozess /opt/mein-zweiter-apache/bin/httpd (apply for host.vars.httpds) (assign where "linux" in host.groups)

This would fix related issues like: #831 

## Current Behavior
see Expected Behavior

## Possible Solution
I fear this will be pretty hard? Currently icinga2 only evaluates this on startup, correct? In our case the host has a custom variable which is used for the apply for rule - so I guess this array could be iterated and one row could be shown for each of the elements

## Steps to Reproduce (for bugs)
Open the list of services shown on a host - the apply rule object is shown but it doesn't really make any sense to use overrides there because they won't be applied because none of the generated objects (using apply for) will be named like this object - see also issue #831 

## Your Environment
* Director version (System - About): master (e148d2523b5c451c37fd23d166a9633d9ed6fd91)
* Icinga Web 2 version and modules (System - About): 2.6.2
* Icinga 2 version (`icinga2 --version`): r2.10.4-1
* Operating System and version: RHEL7
* Webserver, PHP versions: Apache 2.4, PHP 7.1

**Comment by Tqnsls** [comment]
Good idea and imho a step for solving your other issue https://github.com/Icinga/icingaweb2-module-director/issues/1868.
But I also think it's not that simple as the director just builds the config files for icinga2.
